webrender/src/device.rs

```diff
@@ -2051,6 +2055,7 @@ fn gl_texture_formats_for_image_format(format: ImageFormat) -> (gl::GLint, gl::G
                 (gl::RGBA as gl::GLint, GL_FORMAT_BGRA)
             }
         }
+        ImageFormat::RGBA16 => (gl::RGBA16UI as gl::GLint, gl::RGBA_INTEGER),
```

Comment:
We should bring some consistency with names. Having `RGBA8` meaning one thing, `RGBA16 another, and `RGBAF32` on top of it is a bit of a mess. Maybe `RGBA8`, `RGBA16U`. `RGBA32F`?